ext/pdo_pgsql/pgsql_statement.c

```diff
@@ -218,6 +262,16 @@ static int pgsql_stmt_execute(pdo_stmt_t *stmt)
 				}
 			}
 		}
+		if (S->is_unbuffered) {
+			dispatch_result = PQsendQueryPrepared(H->server, S->stmt_name,
+					stmt->bound_params ?
+						zend_hash_num_elements(stmt->bound_params) :
+						0,
+					(const char**)S->param_values,
+					S->param_lengths,
+					S->param_formats,
+					0);
+		} else {
 		S->result = PQexecPrepared(H->server, S->stmt_name,
```

Comment:
identation off

Reply:
I like that the diff shows to make it clear that the code hasn't changed (other than now being conditionned), think "manual way of git diff -w / ticking the Ignore whitespace".
I added a commit dedicated to all reformatting.

Reply:
understood :-) fair enough